ludwig/contribs/mlflow/__init__.py

```diff
@@ -241,7 +249,15 @@ def _log_mlflow(log_metrics, steps, save_path, should_continue, log_artifacts: b
 
     This is used when save_in_background is False.
     """
+    # breakpoint()
     if log_metrics is not None:
+        if log_metrics["llm_eval_examples"] is not None:
+            mlflow.llm.log_predictions(
+                inputs=log_metrics["llm_eval_examples"]["inputs"],
+                prompts=[0] * len(log_metrics["llm_eval_examples"]["inputs"]),
```

Comment:
Based on the example in https://mlflow.org/docs/latest/python_api/mlflow.llm.html, perhaps we should make `inputs=[0] * len(log_metrics["llm_eval_examples"]["inputs"])`  and `prompts=log_metrics["llm_eval_examples"]["inputs"]`. WDYT?

Reply:
Oh wait this is an error. Prompts should actually contain the targets (as per discussion with @geoffreyangus). Let me fix this.